openedx/core/djangoapps/user_authn/views/login.py

```diff
@@ -496,6 +459,9 @@ def login_user(request):
         return response
     except AuthFailedError as error:
         log.exception(error.get_response())
+        if error.error_code == 'inactive-user':
+            error['email'] = user.email
```

Comment:
You need to add this extra data into `error.get_response()`. Look at the third part auth error as an example.